railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -34,7 +34,7 @@ group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   <%- if depends_on_system_test? -%>
   # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '~> 2.13.0'
+  gem 'capybara', '~> 2.13'
```

Comment:
From a glance at https://rubygems.org/gems/capybara/versions, it *sounds* like we might want the more permissive pattern. I don't know whether Capybara has a versioning policy that declares everything below 3.0 to be safe-by-definition, though (e.g. semver).

@twalpole?